tests/e2e/features/artifacts/assetsPage/artifacts-emptyWallet.feature

```diff
@@ -46,3 +46,31 @@ Feature: Artifacts - UI
     #Error state
     Then Element with "partial class" "has-error" should be "visible"
     Then Element with "partial class" "amount-input-error" should be "visible"
+
+  @1489:I
+  Scenario Outline: Check navigation from networks dropdown
```

Comment:
Why do we have Navigation in Artifacts? may be it will be better to move these tests to Artifact folder and call them with an appropriate tag?

Reply:
Moved those tests to appropriate folder - Naviagation.
<img width="250" alt="image" src="https://github.com/matter-labs/dapp-portal/assets/118197764/541969f8-5f58-4750-a267-92ef261c4f39">